destroy.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Why do we need this `destroy.sh` script too?

Reply:
since hes not using docker-compose. hes building with "build.sh" and destroying with "destroy.sh"

- stop both servers
- remove both containers
- remove the image